origin-bridge/util/ipfs.py

```diff
@@ -51,6 +51,9 @@ def directly_pinned_hashes(self, pin_types=None):
         pinned_keys = self.connector.pin_ls()['Keys']
         return [k[0] for k in pinned_keys.items() if k[1]['Type'] in pin_types]
 
+    def pin_json(self, json):
```

Comment:
This method does not pin. It adds json data to IPFS. So let's just call it "add_json" to avoid any confusion.

Reply:
From what I understand any content that is added to the `local` IPFS node is automatically pinned and not garbage collected. Content from other nodes can be garbage collected on the `local` node unless it is pinned explicitly.

Reply:
Yes, it is correct but I still think it is confusing to call this method "pin_json" since a) it is a thin wrapper around the "add_json" method and b) pinning typically refers to pinning a hash :)

Reply:
Yep if we stick to the [IPFS API](https://github.com/ipfs/js-ipfs-api) it should reduce confusion. An add operation is different to a pin operation. You are totally right that an add operation on our node will result in it being pinned though.

Reply:
Good point. Renaming to `app_json` to increase clarity.